constants/Bingo.json

```diff
@@ -1,320 +1,324 @@
+        },
+
+        "Rotten Flesh Collector": {
+            "difficulty": "Medium",
+            "note": [
+                "47.5k Rotten Flesh"
+            ],
+            "guide": [
+                "Unlock the Zombie Minion by killing",
+                "Zombies in the Graveyard in The Hub.",
+                "Place as many minions as you can",
+                "give them a medium storage and the best fuel",
+                "you can. Collect them in time."
+            ]
+        },
+
+        "Fairy Finder": {
+            "difficulty": "Easy",
+            "note": [
+                "25 Fairy Souls"
+            ],
+            "guide": [
+                "Collect those you run along while doing other goals.",
+                "If you have trouble check TimeDeo's fairy soul",
+                "tutorial on YouTube or run /neusouls on to get",
+                "the locations shown."
+
+            ]
+        },
+
+        "Friend Of The Seas": {
+            "difficulty": "Hard",
+            "note": [
+                "Equip an Uncommon or higher Dolphin"
+            ],
+            "guide": [
+                "You have to fish up 250 Sea Creatures",
+                "to get a Common Dolphin. Upgrade it",
+                "with Kat for 100k coins and wait a day.",
+                "You might want to unlock Angler Armor for this."
+            ]
+        },
+
+        "I See Stars": {
+            "difficulty": "Easy",
+            "note": [
+                "5 star a dungeon item"
+            ],
+            "guide": [
+                "Get yourself the Rogue Sword from Jamie",
+                "in The Hub and star it using the Gold Essence",
+                "from the Gold Forger Visitor in your Garden."
+            ]
+        },
+
+        "Trouble is Brewing...": {
+            "difficulty": "Easy",
+            "note": [
+                "Brew any level 4 potion"
+            ],
+            "guide": [
+                "Easiest way of getting this would",
+                "be brewing a Speed 3 potion using",
+                "1 Enchanted Sugar and adding a",
+                "Glowstone Dust from the Builder in The Hub."
+            ]
+        },
+
+        "Spider Queen": {
+            "difficulty": "Medium",
+            "note": [
+                "Summon Arachne"
+            ],
+            "guide": [
+                "To summon Arachne you need to place",
+                "4 Arachne's Callings in the Altar in",
+                "Arachne's Sanctuary. Those drop from slaying",
+                "Arachne's Keepers there. You can get away with",
+                "only placing one by teaming up with others."
+            ]
+        },
+
+        "A Real Head Scratcher": {
+            "difficulty": "Easy",
+            "note": [
+                "Complete Puzzler's Quest"
+            ],
+            "guide": [
+                "Go to the Puzzler in the Dwarven Mines",
+                "and mine the Block he tells you to."
```

Comment:
Might want to mention that Skytils has a solver for it.

Reply:
And Neu has a solver